### PR TITLE
CPP: Fix typo.

### PR DESCRIPTION
**Pull request by geoffw0**
Came across this while reviewing something else.  See `ConvertToVirtualBaseInstruction` a handful of lines below the change for proof that a `ConvertToBaseInstruction` is not necessarily non-virtual.

### CODE REVIEW

File: cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll [hunk 1]
```diff
@@ -974,7 +974,7 @@ class InheritanceConversionInstruction extends UnaryInstruction {
 
 /**
  * Represents an instruction that converts from the address of a derived class
- * to the address of a direct non-virtual base class.
+ * to the address of a direct base class.
```
Comment (dbartol) [inline cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll]:
```suggestion
 * to the address of a base class.
```

Comment (dbartol) [reply cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll]: My typo (or cut-n-paste error) was worse that you thought. In the case of a virtual base class, it's not necessarily even a direct base class.

File: cpp/ql/src/semmle/code/cpp/ir/implementation/raw/Instruction.qll [hunk 2]
```diff
@@ -974,7 +974,7 @@ class InheritanceConversionInstruction extends UnaryInstruction {
 
 /**
  * Represents an instruction that converts from the address of a derived class
- * to the address of a direct non-virtual base class.
+ * to the address of a direct base class.
```
Comment (dbartol) [inline cpp/ql/src/semmle/code/cpp/ir/implementation/raw/Instruction.qll]:
```suggestion
 * to the address of a base class.
```

File: cpp/ql/src/semmle/code/cpp/ir/implementation/unaliased_ssa/Instruction.qll [hunk 3]
```diff
@@ -974,7 +974,7 @@ class InheritanceConversionInstruction extends UnaryInstruction {
 
 /**
  * Represents an instruction that converts from the address of a derived class
- * to the address of a direct non-virtual base class.
+ * to the address of a direct base class.
```
Comment (dbartol) [inline cpp/ql/src/semmle/code/cpp/ir/implementation/unaliased_ssa/Instruction.qll]:
```suggestion
 * to the address of a base class.
```

File: cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll (hunk 1 above)
Comment (geoffw0) [reply cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll]: Thanks, turns out this observation is potentially relevant in https://github.com/Semmle/ql/pull/2638.

**Comment by geoffw0** [comment]
Thanks, I've never used that script before - done.

**Pull request merged**